.github/workflows/deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
```

Comment:
Did you go through and double check that no sensitive information is printed or logged as part of the deploy? It's something we're going to need to be vigilant about with these github deploys.

Reply:
Discussed with you via slack.